apps/nextjs-website/src/components/organisms/Auth/LoginForm.tsx

```diff
@@ -1,19 +1,149 @@
 'use client';
-import { Authenticator, useAuthenticator } from '@aws-amplify/ui-react';
+import { translations } from '@/_contents/translations';
+import IconLogin from '@/components/atoms/IconLogin/IconLogin';
+import { useAuthenticator } from '@aws-amplify/ui-react';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import {
+  Box,
+  Typography,
+  Stack,
+  Checkbox,
+  Grid,
+  TextField,
+  Divider,
+  Button,
+  Card,
+  FormControlLabel,
+  FormControl,
+  InputLabel,
+  OutlinedInput,
+  InputAdornment,
+  IconButton,
+} from '@mui/material';
+import { Auth } from 'aws-amplify';
+import Link from 'next/link';
 import { redirect } from 'next/navigation';
+import { MouseEvent, useCallback, useState } from 'react';
 
 const LoginForm = () => {
+  const {
+    auth: { login },
+    shared,
+  } = translations;
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [showPassword, setShowPassword] = useState(false);
+
   const { authStatus } = useAuthenticator((context) => [context.authStatus]);
 
   if (authStatus === 'authenticated') {
     redirect('/');
   }
 
+  const handleClickShowPassword = useCallback(
+    () => setShowPassword((show) => !show),
+    []
+  );
+
+  const handleMouseDownPassword = useCallback(
+    (event: MouseEvent<HTMLButtonElement>) => {
+      event.preventDefault();
+    },
+    []
+  );
+
+  const onLogin = useCallback(() => {
```

Comment:
There's another ticket for errors handling: [DEV-1047](https://pagopa.atlassian.net/browse/DEV-1047)

Reply:
is there a reason why it couldn't be done in this PR? is it too big?

I mean, this PR should change the UI, without changing the behaviour. but here we are changing the behaviour of what we have deployed.

Reply:
We don't really have the UI for the errors either, so I'd merge these changes first and then add the error handling when the design will be updated. Isn't that right?

Reply:
okay, IMHO in this PR I'd at least show an error or something just to inform the user that something went wrong (and handle the unhandled promises) and then, since the error handling is tracked in another activity, I think you should start asking questions to Norma (I guess) about the design related to the errors, so then we can start the errors handling activity too.

What do you think?

Reply:
Sure